client/state/sites/selectors/get-store-setup.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns store setup information
+ *
+ * @param  {object}  state  Global state tree
+ * @param  {number}  siteId Site ID
+ * @returns {object} Object with store setup information
+ */
+export default function getStoreSetup( state, siteId ) {
```

Comment:
Please handle this check on the server, see PCYsg-qqr-p2 or PCYsg-qpk-p2 for more info. If you have any questions @Automattic/serenity can help walk you through this.

Reply:
Totally fine though to let the task component query if we to display remaining tasks left.

Reply:
Thank you for your comments @gwwar, the documentation you shared is very helpful. 
In this case, as you mentioned [here](https://github.com/Automattic/wc-calypso-bridge/issues/554#issuecomment-666482555), I should add an endpoint in Jetpack to expose the information I need, am I right?
Also, just to confirm, (please correct me if I'm missing something) in this repo I should create a structure similar to what we [use with the `checklist`](https://github.com/Automattic/wp-calypso/tree/master/client/state/checklist), to handle the data we get from Jetpack, yes?

Reply:
> I should add an endpoint in Jetpack to expose the information I need, am I right?

Correct!

> in this repo I should create a structure similar to what we use with the checklist, to handle the data we get from Jetpack, yes?

I'm not sure I follow what's going on there, but I believe it's a checklist for use outside of of My Home, so likely more complicated than you need. I'd say a better model in your case might be the `ConnectAccountsTask` component. Your task component would have a query component that fetches the data from your new endpoint, and updates appropriately. So the overall idea is: 

* My Home page load calls the `home/layout` endpoint.
* Server determines which view to send back; let's say your view in this case.
* The API returns the list of cards to show (which has your `TASK_SITE_SETUP_CHECKLIST_ECOMMERCE` in the main spot).
* The page renders your card/task, which triggers the query component to fetch the checklist data you require.
* When that comes back, the task is visually updated as required.
* As a user completes tasks, your checklist data should remain your "single source of truth", so that any fresh page reload of My Home would go through the above again and render the correct completion data.

Reply:
Thank you @kwight, now get it 🙂